cilium-cli/install/upgrade.go

```diff
@@ -33,15 +33,22 @@ func (k *K8sInstaller) UpgradeWithHelm(ctx context.Context, k8sClient *k8s.Clien
 		Name:         k.params.HelmReleaseName,
 		Chart:        k.chart, // k.chart was initialized in NewK8sInstaller, based on Version and HelmChartDirectory
 		Values:       vals,
-		ResetValues:  k.params.HelmResetValues,
-		ReuseValues:  k.params.HelmReuseValues,
```

Comment:
This doesn't still look correct to me. The `HelmResetThenReuseValues` parameter should be set here, and then propagated to the helm settings inside the `Upgrade` function invoked below.

Reply:
Made changes as requested.